### PR TITLE
OIM Milestone 1.0-ER1 update

### PR DESCRIPTION
**Pull request by hermfischer**
#### Reason for change
Changes for OIM 1.0-ER1 milestone.

#### Description of change
This milestone incorporates editorial clarifications of OIM error messages.

#### Steps to Test
Run the full set of OIM conformance tests from master in gitLab OIM

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: arelle/plugin/loadFromOIM.py [hunk 1]
```diff
@@ -1188,7 +1194,13 @@ def checkMemberTypes(obj, path, pathParts):
...
                 # process extension
                 for extendedFile in documentInfo["extends"]:
-                    extendedOimObject = loadOimObject(extendedFile, mappedUrl, visitedFiles)
+                    try:
+                        extendedOimObject = loadOimObject(extendedFile, mappedUrl, visitedFiles, extensionChain)
+                    except FileNotFoundError:
+                        error("{}:unresolvableBaseMetadataFile".format(errPrefix),
+                              _("Extending document file not found: %(extendingFile)s, referenced from %(extendedFile)s"),
+                              extendingFile=extendedFile, extendedFile=oimFile)
+                        raise OIMException()
```
Comment (sagesmith-wf) [inline arelle/plugin/loadFromOIM.py:1203]: Should probably copy the message over here also

File: arelle/plugin/loadFromOIM.py [hunk 2]
```diff
@@ -1560,21 +1572,30 @@ def dimChecks():
...
-                                                              _("Table %(table)s %(source)s %(dimension)s period-specifier invalid: %(target)s, url: %(url)s"),
-                                                              table=tableId, source=dimSource, dimension=dimName, target=dimAttr, url=tableUrl)
-                                                    if dimValue not in checkedParams:
-                                                        checkedParams.add(dimValue)
-                                                        if dimValue in ("rowNumber", ) or (dimValue in header and dimValue not in commentColumns) or dimValue in tableParameters or dimValue in reportParameters:
-                                                            checkedDims.add(dimValue)
-                                                        else:
-                                                            potentialInvalidReferenceTargets[dimName] = dimValue
+                                            if isinstance(dimValue, str):
+                                                if dimValue.startswith("$"):
```
Comment (sagesmith-wf) [inline arelle/plugin/loadFromOIM.py:1576]: Can you explain some of these cases? What does it mean to have:

- $ (looks like maybe a date?)
- $$
- text:text (looks like a dimension:member pairing?)

**Review by sagesmith-wf — DISMISSED** [review]

File: arelle/plugin/loadFromOIM.py [hunk 3]
```diff
@@ -2351,12 +2382,18 @@ def checkDim(tblTmpl, dimName, dimValue, *pathSegs):
                             elif dimVal is not None:
                                 dimVal = str(dimVal) # may be int or boolean
                             if dimConcept.isExplicitDimension:
-                                mem = qname(dimVal, namespaces)
+                                mem = qname(dimVal, namespaces)
```
Comment (sagesmith-wf) [inline arelle/plugin/loadFromOIM.py:2385]: Nit: Tons of trailing whitespace here

**Comment by hermfischer** [comment]
Replaced by #681

**Pull request closed without merging**